packages/components/src/navigator/test/index.js

```diff
@@ -350,4 +378,31 @@ describe( 'Navigator', () => {
 		expect( getHomeScreen() ).toBeInTheDocument();
 		expect( getToChildScreenButton() ).toHaveFocus();
 	} );
+
+	it( 'should escape the value of the `path` prop', () => {
+		render( <MyNavigation /> );
```

Comment:
Thank you! Once the main "testing env" is set up, it actually becomes quite a trivial task

Reply:
Any reason for the path value escaping?

Reply:
Since the `path` prop is used as the [value for an HTML attribute](https://github.com/WordPress/gutenberg/blob/f821819e850cb043c300cf7b65fbcd8bea4911b2/packages/components/src/navigator/navigator-button/hook.ts#L51) (part of the focus restoration logic), the value of `path` needs to be a valid value for HTML attributes (meaning it can't contain a series of characters like single/double quotes, spaces, etc).

Therefore, the value of `path` is escaped to make sure that it can be safely applied as an HTML attribute  — and this test checks that behaviour

Reply:
Makes sense, thanks :)